modules/greenbidsRtdProvider.js

```diff
@@ -73,11 +75,17 @@ function createPromise(reqBidsConfigObj, greenbidsId) {
 
 function processSuccessResponse(response, timeoutId, reqBidsConfigObj, greenbidsId) {
   clearTimeout(timeoutId);
-  const responseAdUnits = JSON.parse(response);
-  updateAdUnitsBasedOnResponse(reqBidsConfigObj.adUnits, responseAdUnits, greenbidsId);
+  let responseAdUnits;
+  try {
+    responseAdUnits = JSON.parse(response);
```

Comment:
question: Why did you change from `const` to `let` ? I suggest to stick to const if possible.